src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java

```diff
@@ -397,7 +398,7 @@ void updateTransactionRowsFromWorkflowDocuments(ResultSet rs, PreparedStatement
                     initiatorPrincipalId = document.getInitiatorPrincipalId();
                     documentStatus = document.getStatus();
                     if (document.getDateFinalized() != null) {

```

Comment:
Now that the contents of the block have been updated to use `getFinalizedDate()` instead, should the condition itself also be updated to use it? Sorry for not noticing this sooner.